.idea/runConfigurations/arx_3_7_0_osx_64_main.xml

```diff
@@ -0,0 +1,10 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="arx-3.7.0-osx-64-main" type="JarApplication" factoryName="JAR Application">
```

Comment:
Does it really make sense to include this configuration, which points to a specific build of ARX? I suggest to remove this file. If this is not possible, please explain why.

Reply:
This run configuration is if a user wanted to run a jar in intellij, after having run the ant build - this makes it easier. Of course they can use the other builds to test, but that would be testing using intellij's automake. However, as you point out, because we put the version names directly in the jar, it makes it hard here to specify the jar name. Removing is probably the right option.

Reply:
I agree we can remove this. It does provide some useful configuration details for the developer. Is there a "developer readme" or other developer documentation somewhere where we could add some information about running and/or debugging the jar? E.g. to run and allow for debugging, these parameters are required (but perhaps not obvious): `-Xdebug -XstartOnFirstThread -agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005`

Reply:
I see. I think that there are two options:

(1) Create three such configurations, one for each build of ARX (Windows, Linux/GTK and OSX). The configurations should be independent of a release number (e.g. called "arx-current-release-osx"). If possible, I suggest to extract the current release number from ARX's build.xml (property "arx.version"). If this is not possible, it could be possible to include wildcards or the like into the reference to the jar-files (maybe "$PROJECT_DIR$/jars/arx-*-osx-64.jar" would work?)

(2) Create a developer readme. I would be happy to include such a file anyway. This could be implemented instead of (1) or in addition.

Reply:
I think we should just drop this configuration for now - we could do this - but running a jar is extremely easy with intellij, with a few clicks one can generate this. The more important ones we've already included which allow using native intellij's automake.

Reply:
I see. Sounds good to me. Thanks, Fabian